src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -151,7 +155,17 @@
             shortName = INTERVAL_THRESHOLD_ALLELE_FRACTION_SHORT_NAME,
             optional = true
     )
-    protected double intervalThresholdAlleleFraction = 1;
+    protected double intervalThresholdAlleleFraction = 2;
+
+    @Argument(
+            doc = "Enable use of all copy-ratio--segment breakpoints. " +
+                    "(Default behavior uses only breakpoints from segments not called copy neutral, " +
+                    "if calls are available in output of GATK CNV provided, and none otherwise.)",
+            fullName = USE_ALL_COPY_RATIO_SEGMENTS_LONG_NAME,
+            shortName = USE_ALL_COPY_RATIO_SEGMENTS_SHORT_NAME,
+            optional = true
+    )
+    protected boolean useAllCopyRatioSegments = false;
```

Comment:
Is the called segment file configured to be the input in the workflow Queue scripts?  Not in the version I have....